docker/PythonDockerfileDev

```diff
@@ -22,6 +24,8 @@ RUN pip install -U --no-cache-dir \
         bottle \
 #spacy
         spacy \
+#stanza integration for spacy
+        stanza \
```

Comment:
`spacy_stanza` should be installed as well

Reply:
I propose to remove it, since there are at least two issues with the `spacy_stanza` library I bumped into:
1. https://github.com/explosion/spacy-stanza/issues/70. As I understand this affects quality because of imprecise tokenization. Also, this generate verbose output I could not suppress.
2. `stanza` version [is pinned](https://github.com/explosion/spacy-stanza/blob/b57f3482582535f2f07afb0e4ed01df650925d9d/setup.py#L35) and it is not the latest.

Also, doing lemmatization using stanza directly is straightforward, see https://github.com/simjanos-dev/LinguaCafe/pull/255#issuecomment-2105635003.

The code in this PR should be changed a bit to make it work. Currently, it is broken, since I wanted to check the image size and did not care about usable LinguaCafe at this stage).